cmd/eksctl-anywhere/cmd/root.go

```diff
@@ -51,10 +50,14 @@ func initLogger() error {
 		return fmt.Errorf("failed to create logs folder: %v", err)
 	}
 
-	outputFilePath := filepath.Join(".", "eksa-cli-logs", fmt.Sprintf("%s.log", time.Now().Format("2006-01-02T15_04_05")))
+	timeNow := time.Now().Format("2006-01-02T15_04_05")
+	outputFilePath := filepath.Join(".", "eksa-cli-logs", fmt.Sprintf("%s.log", timeNow))
+	managementClusterState := fmt.Sprintf("management-cluster-state-save-%s", timeNow)
```

Comment:
Can we store this in the mgmt cluster directory? maybe under `mgmt-cluster/cluster-state-backup/` or something?

Reply:
Yeah, I do store it in the management cluster directory. I prepend management cluster directory from clusterctl.go file. https://github.com/aws/eks-anywhere/pull/5113/files#diff-682bd4902d8a65030e531aeae6a0c303cb4c74ce5b469387b51ef56fb604d7aaR182
I will change the folder name to `cluster-state-backup`.